aplazame/aplazame.php

```diff
@@ -493,15 +493,11 @@ public function hookDisplayHeader()
             return false;
         }
 
-        $aplazameJsUri = getenv('APLAZAME_JS_URI') ? getenv('APLAZAME_JS_URI') : 'https://cdn.aplazame.com/aplazame.js';
+        $aplazameJsUri = getenv('APLAZAME_JS_URI') ? getenv('APLAZAME_JS_URI') : 'https://cdn.aplazame.com/aplazame.js?public_key='
```

Comment:
You can use https://www.php.net/manual/en/function.http-build-query.php for to create the query string

Reply:
`host` attribute is missed from the query param.

Reply:
😉👍🏻